cmd/nodepool/create.go

```diff
@@ -29,13 +29,15 @@ func NewCreateCommand() *cobra.Command {
 		ClusterName:  "example",
 		NodeCount:    2,
 		ReleaseImage: "",
+		UpgradeType:  "Replace",
 	}
 
 	cmd.PersistentFlags().StringVar(&opts.Name, "name", opts.Name, "The name of the NodePool")
 	cmd.PersistentFlags().StringVar(&opts.Namespace, "namespace", opts.Namespace, "The namespace in which to create the NodePool")
 	cmd.PersistentFlags().Int32Var(&opts.NodeCount, "node-count", opts.NodeCount, "The number of nodes to create in the NodePool")
 	cmd.PersistentFlags().StringVar(&opts.ClusterName, "cluster-name", opts.ClusterName, "The name of the HostedCluster nodes in this pool will join")
 	cmd.PersistentFlags().StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The release image for nodes. If empty, defaults to the same release image as the HostedCluster.")
+	cmd.PersistentFlags().StringVar(&opts.UpgradeType, "upgrade-type", opts.UpgradeType, "The upgrade type is a type of high-level upgrade behavior nodes in a NodePool. Supported options: Replace, InPlace")
```

Comment:
thanks! can you please default to Replace so we don't break current behaviour?

Reply:
Same comment as above: You could default this inline too.